imports/api/settings.js

```diff
@@ -74,6 +75,10 @@ Meteor.methods({
     if (this.userId) {
       let user = Meteor.users.findOne({_id: this.userId})
       if (user.hasGreaterRole(ROLES.admin)) {
+        if ((settings.email !== Settings.findOne().email) && process.env.QLICKER_MAIL_PASSWORD) {
+          const address = Settings.findOne() ? Settings.findOne().email : ''
+          process.env.MAIL_URL = 'smtp://' + address + ':' + process.env.QLICKER_MAIL_PASSWORD + '@smtp.mailgun.org:587'
```

Comment:
Default "from" address should just be admin@$ROOT_URL if not set in the settings. $ROOT_URL is an env variable that is set and that meteor has access to.